### PR TITLE
Add empty skynet plan so release pipelines will stop failing

### PR DESCRIPTION
**Pull request by aaronlademann-wf**
self explanatory

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: Nothing
+description: We must have at least one skynet plan to make Workiva release pipelines happy.
+image: drydock.workiva.net/workiva/skynet-images:node-test-image-latest
+
+scripts:
+  - echo "I have made RM happy"
```
Comment (KealJones) [inline skynet.yaml:6]: #nit
```suggestion
  - print "I have made RM happy"
```

im just kidding it literally makes 0 difference.

**Review by KealJones — APPROVED** [review]
+1

**Comment by aaronlademann-wf** [comment]
@Workiva/release-management-pp

**Comment by aaronlademann-wf** [comment]
@Workiva/release-management-pp

**Comment by KealJones** [comment]
+1 
@Workiva/release-management-p

**Review by KealJones — APPROVED** [review]
+1

**Comment by KealJones** [comment]
+10

**Comment by KealJones** [comment]
@rmconsole-wf

**Review by kealjones-wk — APPROVED** [review]
+1
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**